packages/cli/default/default.yml

```diff
@@ -40,6 +63,74 @@ dbConnection:
       entities:
         $require: '@veramo/data-store?t=object#Entities'
 
+# Server configuration
+server:
+  baseUrl:
+    $ref: /constants/baseUrl
+  port:
+    $ref: /constants/port
+  use:
+    # CORS
+    - - $require: 'cors'
+
+    # Add agent to the request object
+    - - $require: '@veramo/remote-server?t=function#RequestWithAgentRouter'
```

Comment:
Does this have to be specified always by config?
Can this be made default somehow?